client/src/components/common/index.js

```diff
@@ -3,8 +3,8 @@ import Select from './Select';
 import Heading from './Heading';
 import LabelItem from './LabelItem';
 import RowCol from './RowCol';
-import InfoCard from './InfoCard';
+import CardWithImage from './CardWithImage';
 
 export {
```

Comment:
You can remove this export and directly export the files. Check this answer, https://stackoverflow.com/a/35665938

Reply:
Will make the changes. Thanks!